src/js/Content/Features/Community/FriendsAndGroups/CFriendsAndGroups.js

```diff
@@ -0,0 +1,32 @@
+import {CommunityUtils, ContextType, Messenger} from "../../../modulesContent";
+import {CCommunityBase} from "../CCommunityBase";
+import FFriendsSort from "./FFriendsSort";
+import FInviteFriendsToGroup from "./FInviteFriendsToGroup";
+import FGroupsManageButton from "./FGroupsManageButton";
+import FGroupsSort from "./FGroupsSort";
+import {Page} from "../../Page";
+
+export class CFriendsAndGroups extends CCommunityBase {
+
+    constructor() {
+
+        super(ContextType.FRIENDS_AND_GROUPS, [
+            FFriendsSort,
+            FInviteFriendsToGroup,
+            FGroupsManageButton,
+            FGroupsSort,
+        ]);
+
+        this.myProfile = CommunityUtils.currentUserIsOwner();
+
+        Page.runInPageContext(() => {
+            window.SteamFacade.jq(document).ajaxSuccess((event, xhr, settings) => {
+                if (/\/(friends|groups)\/?\?ajax=1$/.test(settings.url)) {
```

Comment:
Maybe parse the URL instead? Using regex can break easily and may not cover everything.

Reply:
I wasn't sure either...I can revert the last commit if that's better.

Reply:
```suggestion
const url = new URL(settings.url);
if (url.searchParams.has("ajax") && (url.pathname.split("/")[3] === "friends" || url.pathname.split("/")[3] === "groups")) {
```